manifests/y/Yandex/Browser/23.1.3.952/Yandex.Browser.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Yandex.Browser

```

```suggestion
PackageIdentifier: Yandex.Yandex
```
WinGet generally follows the naming scheme `Publisher.PackageName` for the PackageID.

---

@mdanish-kh 
I think `Yandex.Browser` would be much more recognizable for users than just `Yandex.Yandex`
`Yandex.Yandex` can be easily confused with just yandex search bar widget or something else.

---

Sure, that isn't a strict rule anyways. But the metadata should be matching `appwiz.cpl` for upgrade matching's sake